apps/frontend/src/components/Class/Ratings/RatingsContainer.tsx

```diff
@@ -0,0 +1,418 @@
+import React, { useEffect, useState } from "react";
+
```

Comment:
I would also just move this file to the `index.tsx` since there's no need to just have the `index.ts` export this file